src/qbProxy.js

```diff
@@ -150,6 +162,13 @@ ServiceProxy.prototype = {
                     detail: jqHXR.responseText
                 };
 
+                if(config.sessionManagement.enable) {
+                    if(errorMsg.code === 401) {
```

Comment:
what does this 'if' check? what case/situation? 
Would be greate to have some comments what is happening here